csbuild/_testing/functional_test.py

```diff
@@ -222,7 +222,7 @@ def RunMake(self, *args):
 		log.SetCallbackQueue(callbackQueue)
 
 		class _shared(object):
-			ret = None
+			ret = None, None, None # type: tuple[int, str, str]
```

Comment:
This is also a weird change. Why did you make this a tuple of Nones? If it's related to the type comment you added (not sure why you added that), I thought any type could be made None, including tuple types, but if not, you can do

```
from typing import Optional, Tuple
# type: Optional[Tuple[int, str, str]]
```

Reply:
Pylint has gotten pretty good at determine object usage and in this case, it was flagging the usage of `_shared.ret` not being unpackable.  The only way to fix it was to set the initial value as a tuple of None.

Reply:
I really, really don't like that. Checking if things are None is something we should be able to do. If pylint can't handle initializing to None with a type comment, I would rather disable the pylint check on that than end up in a situation where we're never allowed to use None as a sentinel value.